cypress/integration/show_property.spec.js

```diff
@@ -54,13 +54,77 @@ describe('Show property', () => {
         )
       })
 
-      it('Display the repairs history for the property', () => {
-        // Repairs history tab should be active
-        cy.get('.govuk-tabs__list-item--selected a').contains('Repairs history')
+      context('Displays the Repairs History for the property', () => {
+        it('Displays the history tab active', () => {
+          cy.get('.govuk-tabs__list-item--selected a').contains(
+            'Repairs history'
+          )
+        })
+
+        it('Displays the history table', () => {
+          cy.get('.govuk-tabs').within(() => {
+            cy.get('.govuk-tabs__tab').contains('Repairs history')
+            cy.get('.govuk-heading-l').contains('Repairs history')
+
+            // Repairs history table headers
+            cy.get('.govuk-table').within(() => {
+              cy.contains('th', 'Reference')
+              cy.contains('th', 'Date raised')
+              cy.contains('th', 'Status')
+              cy.contains('th', 'Description')
+            })
+            // Repairs history table rows
+            cy.get('[data-ref=10000040]').within(() => {
+              cy.contains('10000040')
+              cy.contains('22 Jan 2021')
+              cy.contains('11:02 am')
+              cy.contains('In progress')
+              cy.contains('An emergency repair')
+            })
+            cy.get('[data-ref=10000037]').within(() => {
+              cy.contains('10000037')
+              cy.contains('21 Jan 2021')
+              cy.contains('4:46 pm')
+              cy.contains('Work complete')
+              cy.contains('A very urgent repair')
+            })
+            cy.get('[data-ref=10000036]').within(() => {
+              cy.contains('10000036')
+              cy.contains('21 Jan 2021')
+              cy.contains('4:41 pm')
+              cy.contains('Work complete')
+              cy.contains('An immediate repair')
+            })
+            cy.get('[data-ref=10000035]').within(() => {
+              cy.contains('10000035')
+              cy.contains('21 Jan 2021')
+              cy.contains('3:03 pm')
+              cy.contains('In progress')
+              cy.contains('A normal repair')
+            })
+          })
+        })
+
+        it('Displays the load more button', () => {
+          cy.get('.page-navigation').within(() => {
+            cy.contains('Load more')
+          })
+          // Run lighthouse audit for accessibility report
+          cy.audit()
+        })
+
+        it('Displays more repairs after clicking Load more button', () => {
```

Comment:
I like this test.

But can we delete the test above it? It looks to me like this does everything that one does if we add a wrapping function around our clicking of 'Load more'

```
    cy.get('.page-navigation').within(() => {
      cy.contains('Load more').click()
    })
```